src/base-commands/base-command.js

```diff
@@ -50,39 +48,42 @@ class BaseCommand extends Command {
   }
 
   sanitizeDateString(propertiesVal) {
-    var shortDate = propertiesVal.slice(4, 33);
-    return shortDate;
+    return propertiesVal.slice(4, 33);
```

Comment:
It might be useful here to have a comment that explains what we are slicing for. (e.g. a sample string input/output)